src/services.js

```diff
@@ -70,7 +70,7 @@ const services = {
       name,
       persist: value.persist || false,
       stopTimeSecs: services.getStopTimeSecs(cfg, value),
-      args: command.get(_.merge(value, { rmOnShutdown: cfg.rmOnShutdown }), name, null)
+      args: command.get(_.merge(value, { rmOnShutdown: cfg.rmOnShutdown, privileged: cfg.privileged }), name, null)
```

Comment:
this would use `false` by default if no `privileged` prop is in the config.

Reply:
4da55e2b6313768c9c5426f5c22bffeeac9a747a how's thtat